query/src/main/java/org/infinispan/query/impl/IndexStartupRunner.java

```diff
@@ -1,14 +1,20 @@
 package org.infinispan.query.impl;
 
+import java.lang.invoke.MethodHandles;
+
+import org.hibernate.search.util.common.logging.impl.LoggerFactory;
```

Comment:
You're using HibernateSearch log factory. Is it intentional?

Reply:
right. there are classes of the query module doing the same.... I'm fixing also them